### ISSUE TITLE
Fix docs for downloading files

### ISSUE THREAD

**Issue by ChangyuYan** [ethereum/remix-project#2245]
![Screen Shot 2022-03-26 at 2 41 13 AM](https://user-images.githubusercontent.com/27938057/160233892-9d301c2e-1b49-4e25-907d-f50952189976.png)

As shown in the screenshot above, there is no "Download All Files" option?

**Comment by yann300** [comment]
in which post did you see "Download All Files"? this has been renamed "Download Backup"

**Comment by ChangyuYan** [comment]
It was shown [here](https://remix-ide.readthedocs.io/en/latest/file_explorer.html), second screenshot 

Download backup only gave me a single README.txt file that says: 

```
This is a Remix backup file.
This zip should be used by the restore backup tool in Remix.
The .workspaces directory contains your workspaces.
```

**Comment by ryestew** [comment]
update:https://remix-ide.readthedocs.io/en/latest/file_explorer.html

**Comment by ryestew** [comment]
fix https://github.com/ethereum/remix-project/issues/2245

**Comment by ryestew** [comment]
Thanks for making this issue!

**Comment by jdpalmquist** [comment]
As of September 30th, 2022, I'm still experiencing this issue. 

I see in the comments that someone says it was fixed in #2245... well, guess not since I'm still experiencing it. 

I try to download files developed in Remix IDE and all that downloads is a .zip file that contains a readme. I set the folder options to show hidden files/folders and there is still nothing in there. 

I love this IDE, but don't love the fact that I had to manually copypasta my files out of remix.

**Comment by TestingMyBoi** [comment]
Also having troubles with this, cannot export files from browser. Says "loading files" or whatever then my computer sounds like it's trying to taxi from a runway, then nothing at all happens.

**Comment by mnawalali4** [comment]
I am also facing the same issue

**Comment by jdpalmquist** [comment]
> I am also facing the same issue

I found a work-around of sorts for anyone looking for answers. It's not perfect, but it gets around the issue: use the 'localhost' workspace. (This requires the 'remixd' utility on your system -- npm install remixd)

I downloaded the remixd package from npm and use that, this allows me to specify a folder on my system that remix can use. This way, when I'm writing files on remix, remixd is saving them directly onto my file-system. Then no download feature is needed since the files are there already. 

The command works like this (I'm on linux, command might be a bit different if on windows): 
$>  remixd ./myProject https://remix.ethereum.org

This also works for existing solidity projects you might want to work on that are on your system, just point remixd at the existing project, then open remix IDE and choose the localhost workspace. If you have remixd active, remix will be able to see the directory you have pointed at and presto chango will put all the files in that directory into the remix IDE. (Be aware that remixd will also put a .deps folder that contains contracts imported, like openzeppelin files into it, and it will also make an 'artifacts' folder that will put all the compiled contract artifacts and ABI files into it as well, so you'll have access to the whole lot on your system)

Again, not a perfect solution, but it does work, and in my opinion this is simpler than having to download a package, unzip it, or reupload files back into remix one by one.
